solutions/devsprint-cecilia-saraiva-2/FinanceAppTests/Scenes/FinanceService/NetworkClientStub.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  NetworkClientStub.swift
+//  FinanceAppTests
+//
+//  Created by Henrique Augusto on 16/08/22.
+//
+import Foundation
+
+@testable import FinanceApp
+
+final class NetworkClientSuccessStub: NetworkClientProtocol {
+    let fileName: String
+    var url: URL
```

Comment:
```suggestion
    var url: URL!
```

Reply:
Não entendi o porque do force unwrap. Alias não é necessário neste caso, devido esta variavel estar sendo incializada no construtor 😄

Reply:
É costume meu na verdade de deixar force unwrapping em alguns testes, como o construtor é desnecessario já que a ideia é que a URL que chega no `performRequest` é a que você quer checar, toda vez que o método for checado esse valor vai ser alterado